File: Algorithms/Source Code Algorithms/Sorting/BubbleSort/SCBubbleSort.swift
func sort(_ a: inout [Int]) {
    var i = 1
    while i < a.count {
        for j in 0..<a.count-i {
            if a[j] > a[j+1] {
                let tempAJ = a[j]
                a[j] = a[j+1]
                a[j+1] = tempAJ
            }
        }
        i += 1
    }
}

File: Algorithms/Source Code Algorithms/Sorting/BubbleSort/SCBubbleSort.c
void sort(int *a, int n) {
    for (int i = 1; i < n; i++) {
        for (int j = 0; j < n - i; j++) {
            if (a[j] > a[j+1]) {
                int tempAJ = a[j];
                a[j] = a[j+1];
                a[j+1] = tempAJ;
            }
        }
    }
}
